examples/interacting_with_js_arrays/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+#[macro_use]
+extern crate neon;
+
+use neon::vm::{Call, JsResult};
+use neon::js::{Variant, JsNumber, JsArray};
+
+// https://github.com/dfrankland/node-eyeliner/blob/master/native/src/js_object_utils.rs
+pub fn js_value_to_float(js_number: Variant) -> f64 {
```

Comment:
I wouldn't recommend this way of doing a type check, and the `Variant` type will be going away. I'll show the better style below...